docs/tutorials/hyp3_isce2_burst_stack_for_ts_analysis.ipynb

```diff
@@ -0,0 +1,484 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d20494ef",
+   "metadata": {},
+   "source": [
+    "# Time series analysis with HyP3 and MintPy\n",
```

Comment:
We should give it a different title to distinguish it from the other notebook. Here's a suggestion:
```suggestion
    "# Time series analysis with HyP3 InSAR Burst Products and MintPy\n",
```